optihood/entities.py

```diff
@@ -23,3 +23,215 @@ class CsvInputFilePathsRelative(_enum.StrEnum):
     storages = "storages.csv"
     stratified_storage = "stratified_storage.csv"
     profiles = "profiles.csv"
+
+
+class BusesLabels(_enum.StrEnum):
+    label = "label"
```

Comment:
There is quite a bit of repetition here.
Perhaps, these can be combined somehow.